src/Analysim.Web/Controllers/AccountController.cs

```diff
@@ -84,6 +84,29 @@ public IActionResult GetUserByID([FromRoute] int id)
             });
         }
 
+        /*
+        * Type : GET
+        * URL : /api/account/isadmin/
+        * Description: check the user is admin or not
+        * Response Status: 200 Ok, 404 Not Found
+        */
+        [HttpGet("[action]/{username}")]
+        public IActionResult IsAdmin([FromRoute] string username)
+        {
+            var user = _dbContext.Users
+                .SingleOrDefault(u => u.UserName == username);
+            if (user == null) return NotFound(new { message = "User Not Found" });
+
+            var admins = _configuration
+                .GetSection("AdminUsers")
```

Comment:
Please add an example to the README so we know how to add this to the configuration.